bundle/manifests/siteconfig.clusterserviceversion.yaml

```diff
@@ -801,6 +801,723 @@ metadata:
             ]
           }
         },
+        {
+          "apiVersion": "siteconfig.open-cluster-management.io/v1alpha1",
+          "kind": "ClusterInstance",
+          "metadata": {
+            "name": "example-core-standard",
```

Comment:
Is there anything meaningfully different here than in the example-standard? There really isn't anything uniquely telco or telco-core about this template. I think we should keep the examples here non-telco specific when possible